src/data/fintype/basic.lean

```diff
@@ -1318,6 +1322,14 @@ by rw [←e.equiv_of_fintype_self_embedding_to_embedding, univ_map_equiv_to_embe
 
 namespace fintype
 
+/-- Given `fintype α`, `finset_equiv_set` is the equiv between `finset α` and `set α`. (All
+set on a finite type are finite.) -/
+@[simps] noncomputable def finset_equiv_set [fintype α] : finset α ≃ set α :=
```

Comment:
Should we use
```
lemma finset_equiv_set_symm_apply [fintype α] (s : set α) [fintype s] : finset_equiv_set.symm s = s.to_finset :=
_
```
instead of the autogenerated lemma? This follows the design of, e.g., `finsupp`. Otherwise LGTM. Thanks!
bors d+

Reply:
Sure thing, I've removed `@[simps]` and added the simp lemmas manually.